test/lib/mock-blpapi-wrapper.ts

```diff
@@ -0,0 +1,96 @@
+/// <reference path='../../typings/tsd.d.ts' />
+
+import assert = require('assert');
+import events = require('events');
+import Promise = require('bluebird');
+import _ = require('lodash');
+import BAPI = require('../../lib/blpapi-wrapper');
+
+export interface IInstruction {
+    start?: boolean;    // Default behavior is to start session successfully
+    stop?: boolean;     // Default behavior is to stop session successfully
+    request: string[];
+    //subscribe?: string[];
+    //unsubscribe?: string[];
```

Comment:
I think an additional comment indicating they are not yet implemented might be more warranted. :)